compass_env.yml

```diff
@@ -0,0 +1,19 @@
+name: compass
```

Comment:
Do we need to make a separate conda environment file? 

I did this in dolphin here https://github.com/opera-adt/dolphin/blob/main/conda-env.yml but my reasoning was that I had a `python` version in there, and that fails if you do `pip install -r requirements.txt`.

We have a requirements files that also has python, so right now you could only use that with conda.

Also, if we were to add this conda env for easier installation, should we add `isce3` to it? Seems like we don't want to tell every person they should install it from source like the comment in requirements.txt says.

Reply:
I'm still figuring out if there's any benefit to having the pip install option (where it's usual to have a requirements.txt file that you can link to in your package metadata file) in addition to the conda env file.
For now I agree that just one file would be better, and
- yes I think you should have isce3 on there
- for developers, which we assume are more sophisticated than average if they're trying to get dev isce3 working, they can uninstall the conda one/ go through the build with separate instructions. no need for a separate environment.yaml file with one fewer thing in it.

as a nitpick, I might go for [`environment.yaml` as the file name](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file) or else `conda-env.yaml` to be even more explicit it's for conda. That lets someone know from looking at the file it's associated with conda, since the filename `compass_env.yaml` could be anything

Reply:
I will remove the environment yaml file as it can only be used to create a new environment file. While being slightly more verbose to use, requirements.txt can be used to create a new environment and install packages to an existing one.